internal/sshserver/serverImpl.go

```diff
@@ -510,6 +540,23 @@ func (s *serverImpl) handleGlobalRequests(
 		}
 		requestID := s.nextGlobalRequestID
 		s.nextGlobalRequestID++
+
+		if request.Type == "keepalive@openssh.com" {
```

Comment:
Please implement it as a switch-case and put the components into separate functions. This function will need to be extended in the future. It's better to not create the need for refactoring later if we can avoid it.